tests/zfs-tests/tests/functional/events/zed_fd_spill.ksh

```diff
@@ -0,0 +1,98 @@
+#!/bin/ksh -p
```

Comment:
You'll need to add this test to the `tests/runfiles/linux.run` run file in order for it to by run by the CI.

Reply:
Ah, that's why it didn't fail on GHA! Thanks